styles/controllers/grid/files/icons.less

```diff
@@ -14,36 +14,36 @@
 	/* -- Iconography: http://www.jankoatwarpspeed.com/post/2009/09/17/sixpack-status-icons.aspx --*/
 
 	.pkp_controllers_grid_files_response pkp_controllers_grid_files_success {
-		background: #daf0da url({$baseUrl}/lib/pkp/templates/images/structure/response_success.png);
```

Comment:
I mostly did it so that usage within LESS scripts matches other LESS variables. I'm pretty ambivalent about it too if you want it changed back.

Should we change all LESS variables to camelCase? It's not a format that's very widely used in CSS-land but it would maintain parity across our own platform.